### PR TITLE
Get Started guide on website for BonFHIR + React

### PR DESCRIPTION
**Pull request by ianatacn**
Create documentation the "Get Started" section of the website [bonfhir.dev](https://bonfhir.dev/)

- Add an introduction to BonFHIR
- Add a series of guides including:
   - Start from a Vite template
   - Setup Medplum
   - Fetch and display FHIR data 
   - FHIR search and tables

### CODE REVIEW

File: docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+   return (
+      <FhirQueryLoader query={patientQuery}>
+         {(patient) => (
+            <FhirValue type="HumanName" value={patient.name} />
+         )}
+      <FhirQueryLoader />
+   );
+   ```
+
+6. BonFHIR comes with [Mantine](https://mantine.dev/) components ready to use for developing the application. We can now add additional information about the patient including their birthday, addresses and contact points. `FhirValue` will take care of all data formatting for us.
```
Comment (julienblin) [inline docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md]:
```suggestion
6. BonFHIR components render using [Mantine](https://mantine.dev/) components - so you can mix & match with all Mantine components for layout and data presentation. We can now add additional information about the patient including their birthday, addresses and contact points. `FhirValue` will take care of all data formatting for us. Don't hesitate to play with options as well to explore formatting capabilities.
```

File: docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+   ```tsx
+   <Group>
+      <Text fw={600}>Birthday: </Text>
+      <FhirValue type="date" value={patient.birthDate} />
```
Comment (julienblin) [inline docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md]:
```suggestion
      <FhirValue type="date" value={patient.birthDate} options={{ dateStyle: "full" }} />
```

File: docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      <Paper shadow="xs" p="xl">
+        <FhirQueryLoader query={patientQuery}>
+          {(patient) => (
+            <Stack gap="sm">
+              <Text size="xl">
+                <FhirValue type="HumanName" value={patient.name} />
+              </Text>
+              <Group>
+                <Text fw={600}>Birthday: </Text>
+                <FhirValue type="date" value={patient.birthDate} />
```
Comment (julienblin) [inline docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md]:
```suggestion
                <FhirValue type="date" value={patient.birthDate} options={{ dateStyle: "full" }} />
```

**Review by julienblin — APPROVED** [review]
Thanks a lot -  this is great!
I just added a few nitpicks - fill free to use them or discard if you think it introduces too much complexity at once.

**Comment by julienblin** [comment]
@ianatacn - if you want the PR to build successfully, you'll need to run the following command at the root of the repo:

```shell
pnpm changeset --empty
```

This will create an empty [changeset file in the `.changeset` directory](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md) that needs to be committed as part of the PR.

File: docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md (hunk 2 above)
Comment (ianatacn) [reply docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md]: This is great 👍 A good opportunity to showcase the power of the FHIR value formatters. Thank you!

**Pull request merged**